.markdownlint.yml

```
@@ -0,0 +1,10 @@
+"default": true
+"MD002": false # First heading should be a top level heading
+"MD007": # Unordered list indentation
+  indent: 4
+"MD009": false # Trailing spaces
```

```suggestion
"MD009": true # Trailing spaces
```
just think this is actually helpful as CSS tends to be annoying

---

I'll follow up in a separate PR. There were too many of those. Don't want to have a huge PR.

---

👍